com.unity.formats.fbx/Editor/FbxExportSettings.cs

```diff
@@ -1745,6 +1750,24 @@ internal void Save()
             convertToPrefabSettingsSerialize = ConvertToPrefabSettings.info;
             this.SaveToFile ();
         }
+
+        // Called on creation and whenever the reset button is clicked
+        internal void Reset()
+        {
+            // create presets if none exist
+            if (!m_defaultModelExportSettings) {
+                m_defaultModelExportSettings = new Preset(ExportModelSettings);
```

Comment:
I'm not sure I understand the use of these presets, is it to reset the ExportModelSettings to default? would it work to set `exportModelSettingsSerialize` to null?

Reply:
Yeah, the presets are for resetting `ExportModelSettings` and `ConvertToPrefabSettings` to default. 

Reset is called both on creation and when hitting the reset button, so theyre saved on creation and re-copied over whenever reset is clicked.